server/server.py

```diff
@@ -3,48 +3,70 @@
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
 import os
+import subprocess
 from pathlib import Path
+from urllib.parse import urlparse
 
-from flask import Flask, request
+from flask import Flask, request, send_from_directory
 from flask.json import jsonify
 from werkzeug.utils import secure_filename
 
 
 URLS = []
-ALLOWED_EXTENSIONS = set(["txt", "pdf", "png", "jpg", "jpeg", "gif", "json"])
+ALLOWED_EXTENSIONS = set(["html"])
+
+path = Path("files")
+path.mkdir(exist_ok=True)
 
-Path("files").mkdir(exist_ok=True)
 app = Flask("klaatu_server")
 app.secret_key = "secret key"
-app.config["UPLOAD_FOLDER"] = "files"
+app.config["UPLOAD_FOLDER"] = path.absolute()
 app.config["MAX_CONTENT_LENGTH"] = 16 * 1024 * 1024
+app.config["SERVER_TYPE"] = os.getenv("KLAATU_SERVER_TYPE", "client")
```

Comment:
Not sure if we need to update docs to mention these env vars and possible options.